extensions/notebook/package.json

```diff
@@ -346,7 +346,9 @@
 	},
 	"dependencies": {
 		"@jupyterlab/services": "^3.2.1",
+		"@types/glob": "^7.1.1",
```

Comment:
Its not too be of a deal since we package using webpack, but generally types should be devDependencies not hard dependencies.

Reply:
Removed from dependencies. Thanks!